etc/c/curl.d

```diff
@@ -879,8 +881,10 @@ enum CurlOption {
   /** The CApath or CAfile used to validate the peer certificate
      this option is used only if SSL_VERIFYPEER is true */
   cainfo = 10065,
-  /** 66 = OBSOLETE */
-  /** 67 = OBSOLETE */
+  /** $(UL */
```

Comment:
Does this actually work? I thought macros had to be self-contained within a comment block